src/date-picker/calendar.jsx

```diff
@@ -59,6 +60,7 @@ const Calendar = React.createClass({
       selectedDate: this.props.initialDate,
       transitionDirection: 'left',
       transitionEnter: true,
+      daysArray: [...Array(7)],
```

Comment:
This will be allocation for each instance of the component. Why not simply move outside of the component?